ga4gh/gff3.py

```diff
@@ -0,0 +1,223 @@
+"""
+Object representation of GFF3 data and parser for GFF3 files.
```

Comment:
Then again, https://pypi.python.org/pypi/gff3/0.3.0 has a single, >300 line
deeply nested function that implements all the parsing.  I am not sure I would
want to look at it any more.

Mark Diekhans <markd@soe.ucsc.edu> writes:
> 
> I wrote the original version of this parser and couldn't find one. However now
> I see:
> 
>       https://pypi.python.org/pypi/gff3/0.3.0
> 
> don't know how I missed it.  Worth taking a look at.
> 
> 
> 
> 
> 
> Jerome Kelleher <notifications@github.com> writes:
> > In ga4gh/gff3.py:
> > 
> > > @@ -0,0 +1,223 @@
> > > +"""
> > > +Object representation of GFF3 data and parser for GFF3 files.
> > 
> > High level question here: do we need to implement our own GFF parser? Are there
> > no GFF3 Python packages we can leverage? This is a lot of code we'll need to
> > test pretty hard, so it would be nice if someone else had done it already...
> > 
> > —
> > Reply to this email directly or view it on GitHub.*
> >